packages/api/src/promise/Api.spec.ts

```diff
@@ -26,7 +26,7 @@ describe.skip('Metadata queries', (): void => {
     const api = await ApiPromise.create({ provider: mock, metadata } as unknown as ApiOptions);
 
     expect(api.genesisHash).toBeDefined();
-    expect(api.runtimeMetadata.toJSON()).toEqual(new Metadata(rpcData).toJSON());
+    expect(api.runtimeMetadata).toBeDefined();
```

Comment:
This is with the toJSON() atm is this change - https://github.com/polkadot-js/api/blob/master/packages/api-metadata/src/storage/fromMetadata/createFunction.ts#L120

So the fallback for these (3 of them), are different. I believe it is fine as-is, the parsing of the data is tested extensively.